virtual_images.go

```diff
@@ -216,3 +216,25 @@ func (client *Client) FindVirtualImageByName(name string) (*Response, error) {
 	virtualImageID := firstRecord.ID
 	return client.GetVirtualImage(virtualImageID, &Request{})
 }
+func (client *Client) FindVirtualImageByNameAndType(name string, imagetype string) (*Response, error) {
+        // Find by name, then get by ID
+        resp, err := client.ListVirtualImages(&Request{
+                QueryParams: map[string]string{
+                        "name":       name,
+                        "filterType": "All",
+                        "imageType": "vmware",
```

Comment:
The function includes an imagetype variable but the variable is never used as the imageType query parameter is hardcoded to "vmware".

Reply:
Fixed, I commited my working version instead of the final version. should be fine now